### PR TITLE
8221 - Adjustments for RESUBMIT

### PR DESCRIPTION
**Pull request by andrepestana-aot**
*Issue #:* /bcgov/entity#8221

*Description of changes:*

- Cleared partyIds to make the BE create new applicant records; 
- Changed logic for addComment that was failing for resubmit.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: src/store/actions.ts [hunk 1]
```diff
@@ -485,6 +485,11 @@ export const resubmit: any = async ({ commit, getters }): Promise<boolean> => {
   // override request action code
   nrTypeData['request_action_cd'] = RequestCode.RESUBMIT
 
+  // override applicants partyIds
+  for (let [key, value] of Object.entries(nrTypeData['applicants'])) {
+    value['partyId'] = ''
```
Comment (cameron-freshworks) [inline src/store/actions.ts:490]: What are the id's used for?

**Review by cameron-freshworks — APPROVED** [review]
Looks good to me

**Review by severinbeauvais — APPROVED** [review]
Changes are as we discussed ... looks good.

File: src/store/actions.ts (hunk 1 above)
Comment (andrepestana-aot) [reply src/store/actions.ts:490]: They come as part of the request with the ids for the old NR. In a submission we are creating a new NR and it needs a new applicant record in the DB. With the ids populated it doesn't create new applicants.

**Pull request merged**